tools/debugging/stress_test_transfers.py

```diff
@@ -372,9 +375,11 @@ def do_transfers(
     # be called inside of `propagate_error`.
     current: greenlet = gevent.getcurrent()
 
-    def propagate_error(result: Greenlet) -> NoReturn:
-        current.throw(result.exception)
-        raise RuntimeError("Must not switch back, this greenlet is dead.")
```

Comment:
This was not true at all, `propagate_error` is **not** called from the same greenlet spawned by pool, it is **a new greenlet**, depending on how the callbacks are scheduled it is perfectly fine for this to execute again (which happened, and the extra exception was just polluting even more the script output)